constants.js

```diff
@@ -11,4 +11,6 @@ module.exports = {
 
   ANCHOR_LEFT: 'left',
   ANCHOR_RIGHT: 'right',
+
+  DAY_SIZE: 39,
```

Comment:
`40` was a mistake when I misinterpreted the [`getCalendarMonthWidth` logic](https://github.com/airbnb/react-dates/pull/406/files#diff-7b46ca2b2b498da14bfa494d3106eea5R2). It was causing the missing right padding you pointed out.